ee/fleetd-chrome/src/tables/system_info.ts

```diff
@@ -15,77 +15,50 @@ export default class TableSystemInfo extends Table {
   ];
 
   getComputerName(hostname: string, hwSerial: string): string {
-    const prefix = 'Chromebook'
+    const prefix = "Chromebook";
 
     if (!!hostname?.length) {
-      return hostname
+      return hostname;
     }
 
     if (!!hwSerial?.length) {
-      return `${prefix} ${hwSerial}`
+      return `${prefix} ${hwSerial}`;
     }
 
-    return prefix
+    return prefix;
   }
 
   async generate() {
     // @ts-expect-error @types/chrome doesn't yet have instanceID.
     const uuid = await chrome.instanceID.getID();
 
     // TODO should it default to UUID or should Fleet handle it somehow?
-    let hostname = "";
-    try {
-      // @ts-expect-error @types/chrome doesn't yet have the deviceAttributes Promise API.
-      hostname = (await chrome.enterprise.deviceAttributes.getDeviceHostname()) as string;
-    } catch (err) {
-      console.warn("get hostname:", err);
-    }
+    // @ts-expect-error @types/chrome doesn't yet have the deviceAttributes Promise API.
+    const hostname = (await chrome.enterprise.deviceAttributes.getDeviceHostname()) as string;
```

Comment:
Isn't this the kind of table with multiple "APIs to columns mapping" where we do want to try/catch?

https://github.com/fleetdm/fleet/pull/12441#discussion_r1240170375